_translator-files/po/fr/Installation-for-Macintosh.po

```diff
@@ -154,7 +153,7 @@ msgstr "Jetez un coup d'œil à la"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:42
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```

Comment:
Is adding the space between `{` and `:` correct?

Reply:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but id doesn't apply here) and I didn't corrected it. I removed it.